lib/MessageReader.js

```diff
@@ -52,6 +52,14 @@ class StandardTypeReader {
     return this.view.getUint8(this.offset++);
   }
 
+  typedArray(len, ArrayType) {
```

Comment:
Since this is an internal class it's not a big deal, but I wanted to point out you cannot use this to parse typed arrays of more than 1 byte per element due to platform specific encoding of wider numbers and byte alignment within the array buffer preventing you from constructing typed arrays starting at offsets into the underlying array buffer which are not divisible by the byte length.  e.g. `new Int16Array(arrayBuffer, 10, 4)` will throw an error saying the offset is not divisible by 4.

Reply:
In that case, I guess the BYTES_PER_ELEMENT is not necessary :)

Reply:
Cool, I'll take it out